pyiceberg/catalog/__init__.py

```diff
@@ -257,6 +257,12 @@ def delete_data_files(io: FileIO, manifests_to_delete: List[ManifestFile]) -> No
                 deleted_files[path] = True
 
 
+def _property_as_bool(properties: Dict[str, str], property_name: str, default: bool) -> bool:
```

Comment:
Move this out from `PropertyUtil` since we do not want to add new public util in a patch release